### PR TITLE
test: fix cartridge test

### LINKED ISSUE #128: Tests: cleanup running tarantool instances if test failed.
At the moment, if the cartridge tests fail (test/integration/cartridge/test_cartridge.py), then running instances of tarantool remain.
Before restarting the tests, you must manually shut down these services. It is necessary to automatically shut down tarantool instances in case of test failures. It is also necessary to check the rest of the integration tests for the presence of this problem.

### PR DESCRIPTION
**Pull request by better0fdead**
Added stop of instance after finish of the test, if instance is still running.
Also looked through other test to check for the same behavior.

Closes #128

### CODE REVIEW

File: test/integration/cartridge/test_cartridge.py [hunk 1]
```diff
@@ -2,12 +2,29 @@
...
+import pytest
+
 from utils import run_command_and_get_output, wait_file
 
+cartridge_name = "test_app"
+
+
+@pytest.fixture(autouse=True)
+def stop_cartridge_app(tt_cmd, tmpdir):
+    # Run test
```
Comment (0x501D) [inline test/integration/cartridge/test_cartridge.py:14]: Missing dot at the end of a sentence.

File: test/integration/cartridge/test_cartridge.py [hunk 2]
```diff
@@ -2,12 +2,29 @@
...
+    yield
+    # Stop instance if it is still running
```
Comment (0x501D) [inline test/integration/cartridge/test_cartridge.py:16]: Missing dot at the end of a sentence.

File: test/integration/cartridge/test_cartridge.py (hunk 1 above)
Comment (better0fdead) [reply test/integration/cartridge/test_cartridge.py]: Added

File: test/integration/cartridge/test_cartridge.py (hunk 2 above)
Comment (better0fdead) [reply test/integration/cartridge/test_cartridge.py]: Added

**Pull request merged**